easybuild/easyconfigs/s/Simple-DFTD3/Simple-DFTD3-0.7.0-foss-2022a.eb

```diff
@@ -0,0 +1,47 @@
+# A. Domingo (Vrije Universiteit Brussel)
+# J. Sassmannshausen (Imperial College London/UK)
+
+easyblock = 'CMakeNinja'
+
+name = 'Simple-DFTD3'
+version = '0.7.0'
+
+homepage = 'https://dftd3.readthedocs.io'
+description = """Reimplementation of the D3 dispersion correction. The s-dftd3 project aims to
+provide a user-friendly and uniform interface to the D3 dispersion model and
+for the calculation of DFT-D3 dispersion corrections."""
+
+toolchain = {'name': 'foss', 'version': '2022a'}
+toolchainopts = {'usempi': False, 'openmp': True, 'pic': True}
+
+github_account = 'dftd3'
+source_urls = [GITHUB_LOWER_SOURCE]
+sources = ['v%(version)s.tar.gz']
+checksums = ['19400a176eb4dcee7b89181a5a5f0033fe6b05c52821e54896a98448761d003a']
+
+builddependencies = [
+    ('CMake', '3.23.1'),
+    ('Ninja', '1.10.2'),
+]
+
+dependencies = [
+    ('mctc-lib', '0.3.1'),
+    ('mstore', '0.2.0'),
+    ('TOML-Fortran', '0.3.1'),
+]
+
+build_shared_libs = True
+
+configopts = '-DWITH_BLAS=1 -DWITH_OpenMP=1'
+
+test_cmd = 'ctest'
+runtest = ''
```

Comment:
Could you add a comment explaining the need for this?

Reply:
fixed in https://github.com/easybuilders/easybuild-easyconfigs/pull/16760/commits/a44e30ce8513cf5ff02978fb426c5f970c16ac73